lib/types/models.ts

```diff
@@ -169,6 +170,7 @@ export interface Application {
 	is_archived: boolean;
 	is_discoverable: boolean;
 	is_stored_at__repository_url: string | null;
+	public_organization: NavigationResource<PublicOrganization>;
```

Comment:
I think that this is actually an OptionalNavigationResource b/c a if the org that owns the app has no public app, then the expand here will return an empty array :thinking: